protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -137,49 +156,86 @@ impl<'decoder> SetupConnection<'decoder> {
         }
     }
 
+    /// Checks whether passed flags indicate that the downstream requires standard job.
     pub fn requires_standard_job(&self) -> bool {
         has_requires_std_job(self.flags)
     }
 }
 
+/// Checks if the flags include the Requires Standard Job flag.
 pub fn has_requires_std_job(flags: u32) -> bool {
     let flags = flags.reverse_bits();
     let flag = flags >> 31;
     flag != 0
 }
+
+/// Checks if the flags include the Version Rolling flag.
```

Comment:
```suggestion
/// Helper function to check if `SetupConnection` has `REQUIRES_VERSION_ROLLING` bit flag.
```

Reply:
Those functions dont know  anything about `SetupConnection` really, they just receive the flags